test/parallel_exec.jl

```diff
@@ -1463,3 +1463,18 @@ syms = setup_syms(3, workers())
 clear!(syms, workers())
 test_clear(syms, workers())
 
+# Test partial recovery from a deserialization error in CapturedException
+try
+    expr =
+    quote
```

Comment:
this would more be more obviously the value of expr if it were indented, and maybe start with `expr = quote` on the same line